### DISCUSSION TITLE
Convert spark df to splink df

### DISCUSSION THREAD

**Discussion by guylissak** [moj-analytical-services/splink#2194] [Q&A] [answered]
I have a use case where i want to read my predictions from parquets and then filter it and apply clustering on it, hence i need to convert the spark df to splink df, is this supported?

**Comment by aymonwuolanne** [comment, marked as answer]
Yep, you can use `df_predict = linker.register_table(<your_spark_df>, "predictions")` - see the documentation [here](https://moj-analytical-services.github.io/splink/linker.html#splink.linker.Linker.register_table)
